src/core/store.ts

```diff
@@ -898,3 +898,31 @@ export const createStore = (
 }
 
 export type Store = ReturnType<typeof createStore>
+
+export const createStoreForExport = (
```

Comment:
I wonder if this should be named `createStoreForExporting`, `createStoreToExport` or something else.

Reply:
We can rename the old function because it was private, right?

then we can rename Store -> InternalStore
createStore -> createInternalStore
And finally createStore can be the public name?

Reply:
This should probably have an explicit return type so that changes to internal function return types won’t break the new public API

Reply:
This name is still private and we can change it at any time.
I know what you mean, but let's not change the other part while this is unstable.

Typing is tricky. Let me know if you have trouble with it.